index.ts

```diff
@@ -121,15 +121,18 @@ type WatchableElements =
 	| HTMLTextAreaElement
 	| Iterable<HTMLTextAreaElement>;
 
-export function watch(elements: WatchableElements): void {
+export function watch(
+	elements: WatchableElements,
+	signal?: AbortSignal,
+): void {
 	if (typeof elements === 'string') {
 		elements = document.querySelectorAll(elements);
```

Comment:
I don't expose an explicit `unwatch/removeEventListener` because this selection could return a different set of elements. For example:

```js
watch('textarea');
$('textarea').remove();
unwatch('textarea'); // It wouldn't affect any fields
```